tests/unit/test_config.py

```diff
@@ -114,7 +114,7 @@ def test_xfunction_imports(
         """
         flow_file.write_text(flow_config)
         workflow_config = WorkflowConfig(
-            workflow="name_a_tree", fpath=flow_file, options=Mock(spec=[]),
+            workflow="name_a_tree", fpath=flow_file, options=SimpleNamespace(spec=[]),
```

Comment:
These tests were written with `Mock`, but none of them were actually relying on mocking in any way so converted them to `SimpleNamespace` which does the same thing, but with less unnecessary extras.

Reply:
In that case, might as well be

```suggestion
            workflow="name_a_tree", fpath=flow_file, options=None,
```

or at least need to remove the `spec` arg if using `SimpleNamespace`

```suggestion
            workflow="name_a_tree", fpath=flow_file, options=SimpleNamespace(),
```

(Btw `Mock` is intended be used for stubbing too, it doesn't necessarily have to be used as a full-blown mock - https://docs.python.org/3/library/unittest.mock.html#the-mock-class)